### PR TITLE
Pbi 08 create response test

### PR DESCRIPTION
**Pull request by aryafchandra**
**Describe your changes**

Redo model on response test creation, previously the sender attribute is an assessor, now the sender attribute is an input from textfield

**Acceptance criteria**

- Assessor can create response test
- The system must reject request when sender field is empty
- The system must reject request when subject field is empty
- The system must reject request when prompt field is empty

**Request**
Creating response test
URL: https://localhost:8000/assessment/create/response-test/
Request type: POST
Request data: name, description, sender, prompt, subject
`{
    "name":"Communication Task 3",
    "description":"Communication",
    "subject":"Contact stakeholder",
    "prompt":"Hi! can you contact him asap?",
    "sender":"Director of HR"
}`

### CODE REVIEW

File: assessment/services/assessment.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def validate_response_test(request_data):
         raise InvalidResponseTestRegistration('Prompt Should Not Be Empty')
     if len(request_data.get('subject').split()) == 0:
         raise InvalidResponseTestRegistration('Subject Should Not Be Empty')
+    if len(request_data.get('sender').split()) == 0:
```
Comment (johaneschristian) [inline assessment/services/assessment.py:83]: Hello, Arya. 
May I know why do we need a split in the validation? I think its better if we can use something like
`if request_data.get('sender') is not None`  instead. Tell me what you think :)
Thanks!

Comment (indiradr) [reply assessment/services/assessment.py:83]: Hi Jo, if condition be `if request_data.get('sender') is None` wouldn't that allow empty strings too, which wouldn't be allowed in the first place?
Maybe a better suggestion would be just to remove the .split() altogether and check if the string is an empty string or not by `if len(request_data.get('sender')) == 0`. What do you think? 
Thanks!

Comment (johaneschristian) [reply assessment/services/assessment.py:83]: Hello, Indi.
Yes, that would be a correct approach. 
However, this expression can be further simplified by using `if not request_data.get('sender')` instead which I personally used.

Comment (aryafchandra) [reply assessment/services/assessment.py:83]: Hi, Jo and Indi.
I tried to detect whitespace input using the split method. If I only check the length of the string, it's still possible for someone to enter '     ' and get a length result that is more than 0. What do you think?
Thanks.

Comment (johaneschristian) [reply assessment/services/assessment.py:83]: Okay, Arya. No problem. As long as the software is working and no hidden bugs are present, everything is okay :)

**Pull request merged**